models/project-extension/src/category-recommendation-settings/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as random } from './builder';
```

Comment:
We also need a draft model for this one ([reference](https://github.com/commercetools/merchant-center-services/blob/main/packages/settings/lib/schemas/project-extension.graphql#L146))

Reply:
thanks [eb35c21](https://github.com/commercetools/test-data/pull/562/commits/eb35c21a00f2871acc2b8e1fa6da54d027136b31)